src/module/helpers-dice.js

```diff
@@ -176,44 +202,65 @@ const OseDice = {
     result.target = data.roll.thac0;
     const targetActorData = data.roll.target?.actor?.system || null;
 
-    const targetAc = data.roll.target ? targetActorData.ac.value : 9;
-    const targetAac = data.roll.target ? targetActorData.aac.value : 10;
+    const targetAc = data.roll.target ? targetActorData.ac.value : 20;
+    const targetAac = data.roll.target ? targetActorData.aac.value : -20;
```

Comment:
As I noted in #332 I think this is fine and good for now but doesn't resolve the ultimate UI concern of providing a common sense flow for "no target" combat. We still might as well accept this change as-is because we don't have an enhancement change ready, but I'll be looking forward to designing a new flow in accordance with #332 in the future

Reply:
I agree.